cypress/integration/js_sliding_tree.spec.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable strict */
+/* eslint-disable max-len */
+describe(('Tests for sliding tree.'), () => {
+  before(() => {
+    cy.visit('/');
+  });
+
+  it('Fruits block.', () => {
```

Comment:
it message should start from the `should` word or similar

Reply:
done